autonomy/deploy/generators/kubernetes/templates.py

```diff
@@ -252,11 +252,11 @@
         imagePullPolicy: Always
         resources:
           limits:
-            memory: "1512Mi"
-            cpu: "0.5"
+            memory: "{agent_memory}Mi"
+            cpu: "{agent_cpu}"
           requests:
             cpu: "0.5"
-            memory: "1512Mi"
+            memory: "1024Mi"
```

Comment:
Why do we not also support the modification of the requests/reservation in the same PR?

Reply:
Also, we are changing the default here which might have unexpected consequences.

Reply:
Sorry, this was a mistake

Reply:
Reverted